app/interceptors/ExceptionInterceptor.java

```diff
@@ -72,7 +71,7 @@ private int getStatusCode(final Throwable throwable) {
     private String getMessage(final Throwable throwable, final int status) {
         String message = throwable.getMessage();
         if (StringUtils.isBlank(message)) {
-            message = HttpStatus.getStatusText(status);
+            message = Integer.toString(status);
```

Comment:
org.eclipse.jetty.http.HttpStatus.getMessage(int statusCode) works, there's also one in Spring I believe.